include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -5110,7 +5110,7 @@
       "version" : "None"
     },
     {
-      "opname" : "OpFunctionPointerINTEL",
+      "opname" : "OpConstFunctionPointerINTEL",
```

Comment:
I just wanted to check that this is an intentional change.

Reply:
Yes, it is. This Op code was renamed with a new revision of the appropriate spec.